### ISSUE TITLE
Support TOPMed metrics for single SE

### ISSUE THREAD

**Issue by jekwatt** [hgsc-project-managers/pm-utils#19]
INPUT: Exemplar Illumina Breakdown Report
OUTPUT: TMQC metrics with results (PASS or FAIL)

For input report, there are 3 types and any of these can be used.

Category: 2. QC Report
- Illumina Breakdown Report Show All – Dynamic (real-time data, slower)
- Illumina Breakdown Materialized Report (faster)
- Illumina Breakdown Materialized Report – Analysis Short (less columns, fastest)

[Note from Jim]
Several people use these for several reasons. Some wanted faster response times than the Dynamic report could provide.

Show all Dynamic is real-time data, many columns, its response is slower.
Materialized is older data (but old only a few hours max), many columns still, responds faster.
Materialized Analysis Short is older data as above, has less columns (more focused), responds fastest.

**Comment by jekwatt** [comment]
TMQC_COLS	
```
# tmqc	
sample_id	
collection  # derived from COLLECTION_LIST
pf_hq_aligned_q20_bases	
mean_insert_size	
average_coverage	
wgs_het_snp_q	
wgs_het_snp_sensitivity	
per_ten_coverage_bases	
per_twenty_coverage_bases	
q20_bases	
contamination_pct	
# internal qc only	
unique_aligned_gb	
aligned_bases_pct	
chimeric_rate	
# jw_note	
lane_barcode  # '-'.join(flowcell_id, lane_num, index_id)
analysis_finished_date	
results_path	
reference_path  # new
results	
```

**Comment by jekwatt** [comment]
Issue resolved in PR #24
